README.md

```diff
@@ -1,85 +1,73 @@
 # Velero CSI plugins
 
 
-This repository contains _alpha_ plugins for CSI snapshotting via Velero.
+This repository contains Velero plugins for snapshotting CSI backed PVCs using the [CSI _beta_ snapshot APIs][7].
 
-These plugins are considered experimental and should _not_ be relied upon for production use.
+These plugins are currently in beta as of the [Velero 1.4 release][1] and will reach GA shortly after the CSI volumesnapshotting APIs in upstream Kubernetes reach GA.
 
-Kubernetes 1.14 or newer is required for the CSI snapshot feature.
+For a list or prerequisites, please refer to our documentation [here][2]
```

Comment:
- or -> of
- I would also mention that the setup/install instructions are at that link
- period at the end of the sentence.